src/blocks/EventAccordion/EventInfo.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import styled from "styled-components";
+import { shape } from "prop-types";
+import { AttractionsType, ProductsType, VenueType } from "../../components/types";
+
+import Category from "./Category";
+import CategoryItem from "./CategoryItem";
+import RevealAnimation from "./RevealAnimation";
+import Tile from "../Tile";
+
+
+const Wrapper = styled(RevealAnimation)`
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 16px 47px;
+
+  && > div:nth-child(n + 2) { margin-left: 16px; }
+`
+
+const EventInfo = ({
+  isOpen, 
+  items: {
+    attractions = null,
+    products = [],
+    venue: {
+      city,
+      name: venueName,
+      state,
+      venueUrl,
+      __typename: venueTypeName
+    }
+  }
+}) => {
+  const lineup = attractions ? attractions.slice(0, 3) : null;
+
+  return (
+    <Wrapper className={isOpen ? "expand" : "collapse"}>
```

Comment:
Check out the classNames library. It is super helpful for conditionally applying classNames in circumstances like this